homeassistant/components/alarm_control_panel/verisure.py

```diff
@@ -19,7 +19,7 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Verisure platform."""
     alarms = []
-    if int(hub.config.get('alarm', '1')):
+    if int(hub.config.get(CONF_ALARM, 1)):
```

Comment:
The old code expected 1 or 0 as config values, so this is old behavior: https://home-assistant.io/components/verisure/

I agree that this should eventually be updated. However I don't think that the voluptuous PRs are the place for this.